runtime-modules/content/src/lib.rs

```diff
@@ -1665,7 +1665,7 @@ decl_module! {
 
                     (
                         updated_nft,
-                        RawEvent::BidMadeCompletingAuction(participant_id, video_id, None),
+                        RawEvent::BidMadeCompletingAuction(participant_id, video_id),
```

Comment:
Those changes shouldn't be included here I think, some rebase error?

Reply:
Yes, something weird happened there. I've removed all changes to runtime-modules and chain metadata.